preview/FS-1068-open-type-declaration.md

```diff
@@ -36,43 +34,26 @@ Type providers can provide static classes, hence this would allow type providers
 
 Additionally, important C# DSL APIs are starting to appear that effectively require this.
 
-## Background: static classes
-
-In .NET, "static classes" are abstract and sealed, containing only static members.
-
-To declare these in F#, you need to specify those attributes on a type:
-
-```fsharp
-[<AbstractClass; Sealed>]
-type C =
-    static member M(x: int) = x * 2
-
-C.M(2) |> ignore
-```
-
-A benefit to using these over module-bound functions is that you can take advantage of overloading and optional parameters. F# tooling will also provide more information in tooltips. The downside is that they are not functions, so you miss out on things like first-class functions (methods cannot take other methods as input, etc.).
-
 ## Detailed design
 [design]: #detailed-design
 
-Opening of static classes allows for treating a static class somewhat as if it were a module with module-bound functions inside of it. So the previous code sample could look like this:
+Opening of types allows for treating a type somewhat as if it were a module with module-bound functions inside of it. For example:
 
 ```fsharp
-[<AbstractClass; Sealed>]
-type C =
+type A() =
     static member M(x: int) = x * 2
 
-open type C
+open type A
 
-M(2) |> ignore
+M(2)
 ```
 
```

Comment:
Needs explanation of the syntax `open type` and how this is a new kind of open declaration.

Reply:
Was the summary not sufficient?

Reply:
There's no formal description of the new syntax form. We'll need that, similar to what was done with `nameof`.

Reply:
I adjusted the test.  It's not very formal still but I think it's what we need for now